bundler.d/service.rb

```diff
@@ -1,3 +1,4 @@
 group :service do
   gem 'puma'
+  gem 'puma-status'
```

Comment:
https://rubygems.org/gems/puma-status/versions/1.0

Reply:
This will need a PR to https://github.com/theforeman/foreman-packaging/tree/rpm/develop to introduce `puma-status`.

Reply:
Isn't `puma-status` a CLI app rather than server app? This does not need to be here at all.

Reply:
> This will need a PR to https://github.com/theforeman/foreman-packaging/tree/rpm/develop to introduce `puma-status`.

I asked  Zach to add puma-status to foreman-packaging.

Reply:
> Isn't `puma-status` a CLI app rather than server app? This does not need to be here at all.

`puma-status` is a separate gem and just a cli app. Are you saying we can add this to `foreman-packaging` and remove this entry from here?

Reply:
That makes sense. If it's ready the state file and not actually hooked into the running server then it won't need to be declared here and we can make it a dependency of the `foreman-service` package.

Reply:
thanks, removed `puma-status` from `bundler.d/service.rb`